pkg/boot/kexec/kexec_load_linux.go

```diff
@@ -85,6 +91,169 @@ func rawLoad(entry uintptr, segments []Segment, flags uint64) error {
 	return nil
 }
 
+// KexecLoad loads the given kernel file as to-be-kexeced kernel with
+// the given ramfs file and cmdline string. It uses the kexec "classic"
+// system call, i.e. memory segments + entry point.
+func KexecLoad(kernel, ramfs *os.File, cmdline string) error {
```

Comment:
This should be in pkg/boot/linux

just like it is for pkg/boot/multiboot

Reply:
Ack. Discussing this in thread: https://github.com/u-root/u-root/pull/2317#discussion_r810164147